backend/in_memory_storage/in_memory_storage.go

```diff
@@ -31,10 +31,48 @@ type inMemStore struct {
 	settings models.Settings
 }
 
-func MakeInMemStore() storage.StorageService {
+func MakeInMemStore(defaultDomain string) storage.StorageService {
```

Comment:
Can you add a brief comment to this function signature to explain what `defaultDomain` is for?

Reply:
👍🏻